packages/contracts-bedrock/test/DataAvailabilityChallenge.t.sol

```diff
@@ -430,6 +477,69 @@ contract DataAvailabilityChallengeProxyTest is DataAvailabilityChallengeTest {
         Proxy proxy = new Proxy(address(this));
         proxy.upgradeTo(address(new DataAvailabilityChallenge()));
         dac = DataAvailabilityChallenge(payable(address(proxy)));
-        dac.initialize(DAC_OWNER, CHALLENGE_WINDOW, RESOLVE_WINDOW, BOND_SIZE);
+        dac.initialize(DAC_OWNER, CHALLENGE_WINDOW, RESOLVE_WINDOW, BOND_SIZE, RESOLVER_REFUND_PERCENTAGE);
+    }
+}
+
+contract DataAvailabilityChallengeResolveGasTest is Test {
+    DataAvailabilityChallenge public dac;
+    uint256 constant challengedBlockNumber = 1;
```

Comment:
This test was added to estimate the fixed and variable cost of resolving a challenge. However the measurements are not precise since i don't think there is a way to include the calldata cost in forge scripts. We'll have to follow up with a typescript (or similar) script to estimate the resolution cost from "outside the blockchain".